packages/ws-worker/src/start.ts

```diff
@@ -1,124 +1,11 @@
 #!/usr/bin/env node
-import yargs from 'yargs';
-import { hideBin } from 'yargs/helpers';
-import createLogger, { LogLevel } from '@openfn/logger';
-
+import createLogger from '@openfn/logger';
 import createRTE from '@openfn/engine-multi';
 import createMockRTE from './mock/runtime-engine';
 import createWorker, { ServerOptions } from './server';
+import { parseArgs } from './cli';
 
-type Args = {
```

Comment:
I was thinking it might be better to include all the types and functions related to argument parsing in the same file. If you see any issues with this approach, we can alternatively export them from the `start.ts` file to the `cli.ts` file.

Please let me know your thoughts.

Reply:
hmm. I'm not sure. I can almost see it, but I think the breakdown of what goes where feels a bit muddy.

In other packages  we use `cli.ts` as the entrypoint, which we don't do here. 

I'll have a think about it and get back to you

Reply:
Sure sir, I've committed rest of the changes. I will be happy to make any additional change.

Reply:
Thank you @SatyamMattoo 

I've been busy here today so I'll have to get back to you next week. Hopefully we can wrap it up quickly from here!